src/main/java/frc/team3128/hardware/NAR_TalonFX.java

```diff
@@ -3,9 +3,14 @@
 import com.ctre.phoenix.motorcontrol.ControlMode;
 import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
 
-public class NAR_TalonFX extends NAR_VexController<WPI_TalonFX>{
+public class NAR_TalonFX extends NAR_Motor<WPI_TalonFX>{
```

Comment:
This is wrong. Motor controllers should not extend from Motor

Reply:
Our naming conventions differ from the ones used in the example code. We effectively made your '775ProBase' class into an enum, and the NAR_Motor should really be named NAR_MotorController. I will be pushing something to clarify this but essentially we will be constructing controllers that are passed the motor type in the factory create() method.